BUILD.bazel

```diff
@@ -246,7 +246,7 @@ genrule(
         "aten/src/ATen/native/native_functions.yaml",
```

Comment:
please remove the dependency from the srcs here

Reply:
Can this be done in a follow up? This just add un-necessary dependency to the bazel build right?

Reply:
Done

Reply:
correct this could have been done in a follow up